docs/supported-metrics.md

```diff
@@ -54,3 +54,48 @@ In Mesmer we support 3 types of metrics:
 | Stream throughput            | counter |
 | Operator throughput          | counter |
 | Operator processing time     | counter |
+
+## ZIO (both Default-JVM-Metrics and others)
+
+| Metric                                | Type    |
+|---------------------------------------|---------|
+| Executor Capacity                     | gauge   |
+| Executor Concurrency                  | gauge   |
+| Executor Dequeued Count               |         |
+| Executor Enqueued Count               |         |
+| Executor Size                         | gauge   |
+| Executor Worker Count                 |         |
```

Comment:
counter?

Reply:
@lgajowy, refactored in this commit:
https://github.com/ScalaConsultants/mesmer/pull/470/commits